vendor/photoshop/OpenColorIO_PS_Context.cpp

```diff
@@ -258,3 +274,19 @@ OpenColorIO_PS_Context::getDefaultTransform(const std::string &device) const
     return _config->getDefaultView( device.c_str() );
 }
 
+
+void
+OpenColorIO_PS_Context::getenv(const char *name, std::string &value)
+{
+#ifdef __APPLE__
+    char *env = std::getenv(name);
+
+    value = (env != NULL ? env : "");
+#else
+    char env[1024] = { '\0' };
+
+    const DWORD result = GetEnvironmentVariable(name, env, 1023);
+
+    value = (result > 0 ? env : "");
+#endif
+}
```

Comment:
The env. access implementation is in several files. Do you think OCIO should have Getenv() & SetEnv() is the public API?

Reply:
I made my own function so I could use GetEnvironmentVariable() on Windows instead of std::getenv(), which was not updating. Would use an OpenColorIO version if there was one. I especially thing the $OCIO variable should be formalized somewhere. You can see I made this to do that:
```
static void getenvOCIO(std::string &value) { getenv("OCIO", value); }
```